alpha/lib/model/MediaServerNode.php

```diff
@@ -90,4 +92,21 @@ public function getSessionType($entryServerNode)
 		return '';
 	}
 
+	public function getPartnerIdUrl(DeliveryProfileDynamicAttributes $da)
```

Comment:
either is an instance mathod that can take $this->getDynamicAttributes() from this, or static function that gets the value in the arguments.

Reply:
normally i would agree since the function does not access any instance specific information currently.
however i feel neither option fits well here. I think adding the dynamic attributes to the serverNode instance is wrong since the server node isn't entry or call specific. making the methods static would fit better but i don't like using static methods and params with abstract classes and inheritance due to the way they are evaluated in php